project-metrics/metrics_service/metrics/dummy.py

```diff
@@ -0,0 +1,36 @@
+"""Dummy/example metric.
+
+This metric is not going to stick around. It will live here temporarily as an
+example of how to implement a metric, and it will allow development of
+surrounding infrastructure (serving REST APIs, Cron job configuration, etc.).
+Once one or two real metrics are implemented, this can be removed.
+
+TODO (rcebulko): Remove once other metrics are in place.
```

Comment:
You can rename this to example.py and change the documentation to explain that this is an example, and leave this file in place

Reply:
Done